languages/payplug-fr_FR.po

```diff
@@ -794,7 +794,7 @@ msgid "payplug_error_wrong_credentials"
 msgstr "L'e-mail ou le mot de passe saisi est incorrect."
 
 msgid "payplug_section_login_forgot_password"
-msgstr "Mot de passe oublié?"
+msgstr "Mot de passe oublié ?"
```

Comment:
why the extra space before the "?"

Reply:
It was asked by Aimèe (Should have a space between the “é” and the “?”) in the wooc-382 ticket